readme.md

```diff
@@ -0,0 +1,226 @@
+
+See [remark][] for info on what the remark ecosystem is.
+
+## When should I use this?
+
+You can use this extension if you want to check (lint) and format markdown
+files from within your editor.
+
+To configure this extension, you define your preferred markdown style in a
+configuration file (`.remarkrc`, `.remarkrc.js`, etc. or in `package.json`).
+This file is picked up by `vscode-remark` and other tools (useful for
+contributors that don’t use VS Code).
+
+The configuration file is also used by [`remark-cli`][remark-cli], which is
+recommended to be used alongside `vscode-remark`, as an npm script and/or in
+CI, to enforce the markdown style.
+
+## Install
+
+[Get it from the VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=xxxx).
+Install through VS Code extensions.
+Search for `remark`.
+
+It can also be installed in VS Code: Launch VS Code Quick Open
+(<kbd>Ctrl</kbd> + <kbd>P</kbd>), paste the following command, and press enter.
+
+```txt
+ext install unifiedjs.vscode-remark
+```
+
+See the
+[extension installation guide](https://code.visualstudio.com/docs/editor/extension-gallery)
+for details.
+
+## Use
+
+This example enables VS Code users that have this extension installed to see
+warnings about markdown style in their editor and the `Problems` pane and to
+format markdown from a command.
+
+Make sure that this extension is installed and then install the CLI and
+plugins:
+
+```sh
+npm install remark-cli remark-preset-lint-recommended remark-preset-lint-consistent remark-toc
+```
+
+Then, add a `remarkConfig` to your `package.json` to configure remark:
+
+```js
+  /* … */
+  "remarkConfig": {
+    "settings": {
+      "bullet": "*", // Use `*` for list item bullets (default)
+      // See <https://github.com/remarkjs/remark/tree/main/packages/remark-stringify> for more options.
+    },
+    "plugins": [
+      "remark-preset-lint-consistent", // Check that markdown is consistent.
+      "remark-preset-lint-recommended", // Few recommended rules.
+      [
+        // Generate a table of contents in `## Contents`
+        "remark-toc",
+        {
+          "heading": "contents"
+        }
+      ]
+    ]
+  },
+  /* … */
+```
+
+> 👉 **Note**: you must remove the comments in the above examples when
+> copy/pasting them, as comments are not supported in `package.json` files.
+
+Now, you can open markdown files in your project, and you’ll see squiggly lines
+and warnings in the `Problems` pane.
+Here’s an example that should produce problems you can use to verify:
+
+```markdown
+1) Hello, _Jupiter_ and *Neptune*!
+```
+
+## Formatting
+
+This extension can format markdown files.
+
+To format a file, first press <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>.
+Search for and select `Format Document With…`.
+Now select `remark`.
+
+To make `vscode-remark` the default formatter for markdown, add the following to
+your `settings.json` (which you can open with <kbd>Ctrl</kbd> + <kbd>,</kbd>):
```

Comment:
No. Going even more in depth is more something to explain in a VS Code guide. These are the kind of things that are built into the editor and people just know.

Reply:
I don’t get why you explain complex key combos in many places while not wanting to explain this.

> Going even more in depth is more something to explain in a VS Code guide.

Then link to it.

> These are the kind of things that are built into the editor

Everything is build into the editor.

> people just know.

Please enlighten me, because I don’t know.

Reply:
I added it anyway, although I feel this is getting really verbose.